pages/release-notes.mdx

```diff
@@ -18,6 +18,17 @@ New releases might affect your existing code, queries or configuration.
 Ensure they're updated to align with the latest updates and changes.
 
 
+### Upgrading to Memgraph v2.21.0
+
+<Callout type="error">
+
+{<h3> Breaking change </h3>}
+
+Coordinator log store needs to be started from scratch.
```

Comment:
@as51340, could you add a bit more details? In other words, what exactly user has to do here, what's going to happen if it's not done, what's going to happen when it's done? 🙏